kibana/Dockerfile

```diff
@@ -43,8 +41,8 @@ ADD kibana.yml ${KIBANA_CONF_DIR}/
 ADD lib/* ${HOME}/
 ADD run.sh utils install.sh prep-install.${RELEASE_STREAM} ${HOME}/
 ADD logo-OCP-console-hdr-thin.svg ${KIBANA_HOME}/installedPlugins/origin-kibana/public/images/logo-origin-thin.svg
-RUN sh ${HOME}/install.sh
-RUN rm -r /tmp/lib
+RUN (which node || ln -s $(which $NODE_BIN) "/usr/bin/node") && \
```

Comment:
I've had problems using `which` in the past since it seems to be in an optional package - use `type -p` instead which is built-in to the shell.

Reply:
+1